packages/serve/src/types.ts

```diff
@@ -0,0 +1,5 @@
+export type devServerOptionsType = {
```

Comment:
It would be ideal to have a full dev server config type once this is worked on: https://github.com/webpack/webpack-cli/issues/866 and once we have finished major dev server options changes

We already have it here but needs to be updated, and I don't want to be importing the `generators` package just for this https://github.com/webpack/webpack-cli/blob/next/packages/generators/src/types/index.ts

Reply:
@Loonride friendly ping

Reply:
Sorry, need to focus on webpack-dev-server v4 first, will get back to this